source/neuropod/bindings/c/README.md

```diff
@@ -0,0 +1,7 @@
+# The C api interface:
+
+1. To build & to unit test:
+```
+bazel build //neuropod/bindings/c:c_api
+bazel test --test_output=errors //neuropod/tests:test_c_api
```

Comment:
FYI, you don't need to run `bazel build` before running `bazel test` (it'll rebuild things as necessary)

Reply:
Yes, this is how I run tests. "build" is when need to produce so. btw, I am not sure if "bazel test" produces .so files really.